api/v1alpha1/mirrorpeer_types.go

```diff
@@ -46,10 +49,18 @@ type PeerRef struct {
 // MirrorPeerSpec defines the desired state of MirrorPeer
 type MirrorPeerSpec struct {
 	// Items is a list of PeerRef.
-
 	// +kubebuilder:validation:MaxItems=2
 	// +kubebuilder:validation:MinItems=2
 	Items []PeerRef `json:"items"`
+	// +kubebuilder:validation:Enum=snapshot;journal
```

Comment:
keep the API changes in a separate commit along with the generated code for API.

Reply:
separated the commits